ludwig/models/modules/sequence_decoders.py

```diff
@@ -149,104 +149,56 @@ class SequenceTaggerDecoder(Layer):
     def __init__(
             self,
             initializer=None,
-            regularize=True,
+            use_bias=True,
+            kernel_initializer='glorot_uniform',
```

Comment:
although Dense uses ths name, throughout the rest of the code we are always calling this weights_initializer. The same  applies to kernel_regularizer too.

Reply:
done